frontend_tests/node_tests/popovers.js

```diff
@@ -241,7 +241,7 @@ test_ui("actions_popover", (override) => {
             case "actions_popover_content":
                 assert.equal(
                     opts.conversation_time_uri,
-                    "http://chat.zulip.org/#narrow/stream/Bracket.20%28.20stream/topic/Actions.20%281%29/near/999",
+                    "http://chat.zulip.org/#narrow/stream/Bracket.20.28.20stream/topic/Actions.20.281.29/near/999",
```

Comment:
It's unclear that that changing the encoding here without changing decoding logic is correct; can you extend this test to call the `hash_util` decoding functions to confirm they handle this correctly?

Reply:
There is no need to change decoding logic because before decoding any URL, we first convert all the `.` to `%.` Please refer to the attached screenshot:
![17572(pull)](https://user-images.githubusercontent.com/70478296/111153282-c37a1780-85b7-11eb-8460-c54d4943b754.png)